.github/actions/get-workflow-job-id/action.yml

```diff
@@ -26,6 +26,20 @@ runs:
         retry_wait_seconds: 30
         command: |
           set -eux
-          python3 -m pip install requests==2.26.0
-          GHA_WORKFLOW_JOB_ID=$(python3 .github/scripts/get_workflow_job_id.py "${GITHUB_RUN_ID}" "${RUNNER_NAME}")
+
+          if [[ -z "${CONDA_RUN:-}" ]]; then
+            # If no conda has been setup, try to use system python
+            python3 -m pip install requests==2.26.0
```

Comment:
As of https://github.com/pytorch/pytorch/pull/92191 requests should no longer be a requirement for the script